scripts/Makefile

```diff
@@ -140,7 +140,7 @@ testsuite:
 # It assumes that some covrage reports already exists, otherwise it will fail
 .PHONY: coverage-report
 coverage-report:
-	python ../libbeat/scripts/aggregate_coverage.py -o ./${COVERAGE_DIR}/full.cov ./${COVERAGE_DIR}
+	python ${GOPATH}/src/github.com/elastic/beats/dev-tools/aggregate_coverage.py -o ./${COVERAGE_DIR}/full.cov ./${COVERAGE_DIR}
```

Comment:
Thx @ruflin, I was aware of that but I'm not sure how frequent should I update libbeat to catch everything (I thought I could update per libbeat release, e.g. 1.0, 1.1, etc.). And, I expect that the next `make update` would overwrite this change, right?